src/features/app/context/guestLocale.js

```diff
@@ -0,0 +1,44 @@
+import React, { createContext, useState, useEffect } from 'react';
+
+export const GuestLocaleContext = createContext();
+
+const STORAGE_KEY = 'RAB_GUEST_LOCALE';
```

Comment:
we used `'session'` for storing the session. maybe we can just use `'guest_locale'` here?